src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyLiPClaimantHwFOutcomeHandler.java

```diff
@@ -73,11 +73,13 @@ public Map<String, String> addProperties(CaseData caseData) {
         };
     }
 
-    private String getTemplate(CaseEvent hwfEvent) {
+    private String getTemplate(CaseEvent hwfEvent, boolean isBilingual) {
         if (emailTemplates == null) {
             emailTemplates = ImmutableMap.of(
                 CaseEvent.NO_REMISSION_HWF,
-                notificationsProperties.getNotifyApplicantForHwfNoRemission()
+                isBilingual ? notificationsProperties.getNotifyApplicantForHwfNoRemissionWelsh()
```

Comment:
I think this is not going to work as we are initializing emailTemplates only once. If map is initialized for bilingual it will always have template for bilingual for all other subsequent calls.